go.work.sum

```diff
@@ -1,172 +1,2040 @@
+4d63.com/gochecknoglobals v0.1.0/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
```

Comment:
Copy this file from main and replace these changes (I think we need to add this to .gitignore, because I have accidentally almost commit this myself yesterday)